src/base/ddl_log.c

```diff
@@ -888,7 +888,7 @@ logddl_write_tran_str (const char *fmt, ...)
 	  len = DDL_LOG_BUFFER_SIZE;
 	}
 
-      if (len < 0 || fwrite (msg, sizeof (char), len, fp) != len)
+      if (fwrite (msg, sizeof (char), len, fp) != len)
```

Comment:
I want to leave len as an **int** not a **size_t** and change it like this (same as line #970):
`if (len < 0 || fwrite (buf, sizeof (char), len, fp) != (size_t) len)`

Reply:
Changed back